src/GitHub.App/ViewModels/GitHubPane/GitHubPaneViewModel.cs

```diff
@@ -213,18 +219,13 @@ public void Dispose()
 
                 connectionManager.Connections.CollectionChanged += (_, __) => UpdateContent(LocalRepository).Forget();
 
-                BindNavigatorCommand(paneServiceProvider, PkgCmdIDList.pullRequestCommand, showPullRequests);
-                BindNavigatorCommand(paneServiceProvider, PkgCmdIDList.backCommand, navigator.NavigateBack);
-                BindNavigatorCommand(paneServiceProvider, PkgCmdIDList.forwardCommand, navigator.NavigateForward);
-                BindNavigatorCommand(paneServiceProvider, PkgCmdIDList.refreshCommand, refresh);
-                BindNavigatorCommand(paneServiceProvider, PkgCmdIDList.githubCommand, openInBrowser);
-
-                paneServiceProvider.AddCommandHandler(Guids.guidGitHubToolbarCmdSet, PkgCmdIDList.helpCommand,
-                     (_, __) =>
-                     {
-                         browser.OpenUrl(new Uri(GitHubUrls.Documentation));
-                         usageTracker.IncrementCounter(x => x.NumberOfGitHubPaneHelpClicks).Forget();
-                     });
+                var menuService = (IMenuCommandService)paneServiceProvider.GetService(typeof(IMenuCommandService));
+                BindNavigatorCommand(menuService, PkgCmdIDList.pullRequestCommand, showPullRequests);
```

Comment:
Is there a reason you're not using the `BindCommand` extension method?

Reply:
`BindNavigatorCommand` does use `BindCommand`. It's just so we don't have to specify `(new CommandID(Guids.guidGitHubToolbarCmdSet, commandId)` for every command.